composables/use-reader.ts

```diff
@@ -1,27 +1,28 @@
-export default function () {
-  const config = useRuntimeConfig()
+export default function (params: {
+  nftClassId?: Ref<string> | string
+} = {}) {
+  const nftClassId = computed(() =>
+    getRouteQuery('nft_class_id') || toValue(params.nftClassId) || '',
+  )
 
-  const nftClassId = computed(() => getRouteQuery('nft_class_id'))
   const bookInfo = useBookInfo({ nftClassId: nftClassId.value })
 
   const nftId = computed(() => {
     const id = getRouteQuery('nft_id')
-    return id !== undefined && id !== '' ? id : bookInfo.firstUserOwnedNFTId.value
+    return id || bookInfo.userOwnedNFTIds.value[0]
```

Comment:
```suggestion
    return id || bookInfo.firstUserOwnedNFTId.value
```

Reply:
[🐛 Update logic for handling download items](https://github.com/likecoin/3ook-com/pull/47/commits/55de6c3a90bd03139a8b75a2fd93135a2746618f)